OMCompiler/Compiler/NFFrontEnd/NFSimplifyExp.mo

```diff
@@ -81,7 +81,7 @@ algorithm
     case Expression.CREF()
       algorithm
         exp.cref := ComponentRef.simplifySubscripts(exp.cref);
-        exp.ty := ComponentRef.getSubscriptedType(exp.cref);
+        exp.ty := ComponentRef.getSubscriptedType(exp.cref, true);
```

Comment:
Looks like this change is responsible for the [the latest regressions](https://libraries.openmodelica.org/branches/history/master/2022-12-16%2012:15:59..2022-12-16%2022:26:10.html). I guess taking the scope into account here isn't correct for the frontend at least.